### ISSUE TITLE
`python/storeTools_cfg.py` `keepOnlyFilesFromGoodRun` error on new files of Run2015C datasets

### ISSUE THREAD

**Issue by xealits** [cms2l2v/2l2v_fwk#139]
Hi all,

I am trying to submit jobs with `/SingleMuon/Run2015C_25ns-16Dec2015-v1/MINIAOD` dataset.
And `keepOnlyFilesFromGoodRun` fails on some files.
And example:

    root://eoscms//eos/cms/store/data/Run2015C_25ns/SingleMuon/MINIAOD/16Dec2015-v1/00000/002C22D4-E1AF-E511-AE8E-001E673971CA.root

The script uses `#FASTER technique only based on run number and file name parsing`
which is based on

    ...
    if '/00000/' in F:
        Fsplit = F.split('/00000/')[0].split('/')
    ...

-- apparently it does not apply here.

The good thing is `Run2015D` datasets are submitted without problems. And they have the bulk of luminosity.

Just thought it is worth reporting.

**Comment by quertenmont** [comment]
yes, that trick doesn't work for rereco samples as they do not follow the same naming convention.

But, excuse me to ask... who are you and which analysis are you working on ?

**Comment by xealits** [comment]
My name is Alex, I work on ttbar->leptontau, I am from LIP CMS group -- the group which Pietro Vischia is from.

**Comment by quertenmont** [comment]
ha ok.

Thanks for reporting.
Loic

2016-03-31 19:18 GMT+02:00 Alex Toldaiev <notifications@github.com>:

> My name is Alex, I work on ttbar->leptontau, I am from LIP CMS group --
> the group which Pietro Vischia is from.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms2l2v/2l2v_fwk/issues/139#issuecomment-204031099>
>



-- 
---------------------------------------------------------
Loic Quertenmont - European Organization for Nuclear Research (CERN) - CMS
experiment
Office CERN   : bldg.40, RB-08
Phone CERN    : +41 2276 71548
Mobile BE     : +32 474 831531
WWW           : http://www.fynu.ucl.ac.be/users/l.quertenmont/
